config/initializers/time_formats.rb

```diff
@@ -0,0 +1,18 @@
+# Time Formats
+# ============
+#
+# Adds named date and time formats for time.to_s and date.to_s
+#
+# To display the current date and time in all the formats in your current environment:
+#
+#   Date::DATE_FORMATS.keys.push(:default).each{|k| puts "#{k}: #{Date.today.to_s(k)}"}
+#   Time::DATE_FORMATS.keys.push(:default).each{|k| puts "#{k}: #{Time.now.to_s(k)}"}
+#
+# For examples time = Time.local(2009,12,4,15,30,27)
+
+shared_formats = {
+  d_month_yyyy: "%-d %B %Y", # '4 December 2009'
```

Comment:
Nit: Found this label somewhat confusing - as in I couldn't tell what the output would be or when I should use it - maybe alt name could be "govuk_date" - as its a standard format in the GOV.UK style guide.

Reply:
Single digit day, word month, four digit year. Happy to change it to `:govuk_date` if it is defined as such in the style guide.